codespell_lib/data/dictionary.txt

```diff
@@ -26129,6 +26132,8 @@ paermission->permission
 paermissions->permissions
 paeth->path
 pagagraph->paragraph
+pagent->pageant
+pagents->pageants
```

Comment:
Perhaps `agents`?  Not sure about it, though it is certainly not a common typo: in GitHub stands for "permanent agent", "pAgent". I don't see anyone unwillingly typing `pagents` instead of `agents`. Your opinion?

Reply:
See above for singular.